proposals/p0162.md

```diff
@@ -274,8 +274,8 @@ declaration:  "fn" identifier tuple return_type '{' statement_list '}'
 
 the `tuple` is used as a pattern to describe the parameters of the function. The
 grammar for function definitions does not currently include
-[implicit parameters](https://github.com/josh11b/carbon-lang/blob/generics-docs/docs/design/generics/terminology.md#implicit-parameter),
-but the intent is to add them to the grammar in the future.
+[implicit parameters](/docs/design/generics/terminology.md), but the intent is
+to add them to the grammar in the future.
```

Comment:
I don't see this implicit parameter section on either carbon-language or jsosh11b's fork. Maybe the link should be removed?

Reply:
The term changed, suggested appropriate anchor tag.